docs/tutorials/transaction-tutorial.md

```diff
@@ -43,15 +52,21 @@ See our [Knex guide](./using-knex.md) for more information.
 <TabItem value="drizzle" label="Drizzle">
 
 ```javascript
-export const greetings = pgTable('greetings', {
+export const GreetingRecord = pgTable('greetings', {
   name: text('name'),
-  note: text('note')
+  note: text('note'),
 });
 
-export class DBOSGreetings {
+export class Greetings {
+  //..
   @Transaction()
   static async insertGreeting(ctxt: TransactionContext<NodePgDatabase>, name: string, note: string) {
-    await ctxt.client.insert(greetings).values({name: name, note: note});
+    await ctxt.client.insert(GreetingRecord).values({name: name, note: note});
+  }
+
+  @Transaction({ readOnly:true })
+  static async getGreetings(ctxt: TransactionContext<NodePgDatabase>): Promise<{name: string | null, note: string | null}[]> {
```

Comment:
Nit: maybe define a type like other ORMs do:

interface GreetingRecord {
  name: string;
  note: string;
}

Reply:
In other ORMs it was a requirement - I had to create the type or else the linter would complain. With Drizzle, it's not really a requirement. Some people may see it as a benefit? So that's why I didn't make one here... WDYT?

Reply:
Don't make one, Drizzle gets its types from your schema (and return types are just whatever you asked for, of course).